### PR TITLE
Website Demo: VGG image classifier

### PR DESCRIPTION
**Pull request by crockpotveggies**
*Work in progress, do not merge*

Interactive demo for the VGG-16 DL4J classifier.

### CODE REVIEW

File: .idea/encodings.xml [hunk 1]
```diff
@@ -11,13 +11,9 @@
     <file url="file://$PROJECT_DIR$/deeplearning4j-scaleout/deeplearning4j-scaleout-akka" charset="UTF-8" />
```
Comment (agibsonccc) [inline .idea/encodings.xml]: Delete this?

File: .idea/misc.xml [hunk 2]
```diff
@@ -1,55 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="MavenBuildProjectComponent">
```
Comment (agibsonccc) [inline .idea/misc.xml]: Delete this?

File: deeplearning4j-keras/dependency-reduced-pom.xml [hunk 3]
```diff
@@ -0,0 +1,189 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (agibsonccc) [inline deeplearning4j-keras/dependency-reduced-pom.xml]: We can get rid of this for gh-pages I believe.

File: .idea/misc.xml (hunk 2 above)
Comment (crockpotveggies) [inline .idea/misc.xml]: This should be removed now.

**Comment by crockpotveggies** [comment]
Should all be good to go.

**Comment by crockpotveggies** [comment]
Results are looking slick with raver's improvements:

screen shot 2017-02-22 at 6 26 18 pm

**Comment by slinlee** [comment]
Nice. Can you clean up the label strings too?  Capitalized and spaces for
the underscores.

**Comment by crockpotveggies** [comment]
Yea I can add some javascript magic to clean it up. That's just the result from the server

**Comment by crockpotveggies** [comment]
done! please merge at your delight

**Comment by crockpotveggies** [comment]
@agibsonccc for some reason your requested changes aren't marking as complete. But when I check the .git tracking on my end, that file is removed.

**Pull request merged**